src/renderer/views/Playlist/Playlist.scss

```diff
@@ -81,23 +128,40 @@
 @media only screen and (width <= 850px) {
   .routerView {
     flex-direction: column;
-  }
 
-  .playlistInfo {
-    box-sizing: border-box;
-    position: relative;
-    inset-block-start: 0;
-    z-index: 1;
-    block-size: auto;
-    inline-size: 100%;
-  }
-
-  .playlistItems {
-    box-sizing: border-box;
-    inline-size: 100%;
+    &.list {
+      .playlistInfoContainer {
+        box-sizing: border-box;
+        position: relative;
+        inset-block-start: 0;
+        block-size: auto;
+        inline-size: 100%;
+      }
+
+      .playlistItemsCard {
+        box-sizing: border-box;
+        inline-size: 100%;
+      }
+    }
   }
 }
 
 .message {
   color: var(--tertiary-text-color);
 }
+
+@media only screen and (width <= 680px) {
+  .routerView{
+    &.grid {
+      margin-block-start: 230px;
```

Comment:
I realize this specific value (as well as the one above) is set because the position of the fixed-top-bar is either `fixed` or `absolute`, and it won't push the body of the page down below it. However, I feel it looks weird in the edge cases of either more than expected or less than expected content. 

![image](https://github.com/FreeTubeApp/FreeTube/assets/106682128/54eb4c60-89e2-4d65-9d1a-f43caaa2f5c3)

![image](https://github.com/FreeTubeApp/FreeTube/assets/106682128/1acc5fd3-eac8-4bc8-a914-0f4affd28a8a)

Reply:
This is unfortunately hard to fix generically. I can truncate the title with ellipses and add a styling case for no videos.